app.css

```diff
@@ -0,0 +1,94 @@
+:root{
+    --primary-color:red;
+    --background-color:#F8FAFC;
+}
+body{
+    margin: 0;
+    padding: 0;
+    background-color: var(--background-color)
+}
+/*Navigation Custom CSS*/
+.navbar{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color: var(--black);
+    color:var(--white);
+    padding: 0.5rem;
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    z-index: 1;
+}
+.nav_right_side{
+    display: flex;
+    justify-content: space-evenly;
+}
+.img_logo{
+    height: 3.125rem;
+    width: 12.5rem;
+}
+/*Custom size of icon for the nav bar*/
+.fa-2x{
+    font-size: 1.6em;
+}
+.main_container{
+    padding-top: 4.5rem;
+    min-height: 85vh;
+    gap: .5rem;
+}
+.home_banner{
+    width: 100%;
+}
+/*Making card fit content*/
+.card{
+    width: fit-content;
+    height: fit-content;
+}
+
+.container_featured_category,.container_games_from_top_brands{
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-around;
+    gap: 0.5rem;
+}
+.container_pick_your_accessories{
+    display: grid;
+    grid-template-columns: repeat(auto-fit,minmax(222px,1fr));
+    gap: 0.4rem;
+}
+.card_featured_category{
+    height: 6.25rem;
+    width: 12.5rem;
+    display: flex;
+    flex-flow: column wrap;
+    justify-content: center;
+    align-items: center;
+    gap: 0.5rem;
+    border-radius: 5px;
+    box-shadow: 0 2px 1px -1px rgb(0 0 0 / 20%),
+    0 1px 1px 0 rgb(0 0 0 / 14%),
+    0 1px 3px 0 rgb(0 0 0 / 12%);
+}
+.link_wrapper{
+    text-decoration: none;
+    color: inherit;
+}
+
+/*Footer Custom CSS*/
+.footer {
+    min-height: 14vh;
+    background-color: black;
+    color: white;
+    padding: 1rem 1rem;
+    display: flex;
+    justify-content: space-between;
+}
+ul{
+    padding-inline-start: 0.5rem;
+ }
```

Comment:
Can shift it to reset.css file where you can pass the list-style-type:none and this property of padding-inline-start

Reply:
done